lib/kubernetes-deploy/cluster_resource_discovery.rb

```diff
@@ -16,8 +16,26 @@ def crds
       end
     end
 
+    def global_resource_names
```

Comment:
nit: this is a list of kinds, not names (which strictly speaking are defined separately)